src/main/java/com/iemr/hwc/repo/login/UserLoginRepo.java

```diff
@@ -0,0 +1,37 @@
+/*
+* AMRIT – Accessible Medical Records via Integrated Technology 
+* Integrated EHR (Electronic Health Records) Solution 
+*
+* Copyright (C) "Piramal Swasthya Management and Research Institute" 
+*
+* This file is part of AMRIT.
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see https://www.gnu.org/licenses/.
+*/
+package com.iemr.hwc.repo.login;
+
+import com.iemr.hwc.data.login.Users;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.data.rest.core.annotation.RestResource;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RestResource(exported = false)
+public interface UserLoginRepo extends CrudRepository<Users, Long> {
+	@Query(" SELECT u FROM Users u WHERE u.userName = :UserName AND u.deleted = false ")
```

Comment:
This method is not required. Join userbiometrics with user table.

Reply:
Using join is more complex to implementation. that's why I create a new user repo class to get the user infos. We have the same result at the end.

Reply:
@roopesh-beehyv , babacar and I had a discussion. If we want to use Join we will need to add a biometric record at the time of user creation only so that we can Join based on userId. In the current implementation, we cannot use that . That's why we require to query the user table first to get the user details and then save that to biometric table.

Reply:
I want to discuss on this. I am merging this for now